### ISSUE TITLE
madsrdf:isMemberOfMADSScheme

### ISSUE THREAD

**Issue by kirkhess** [lcnetdev/marc2bibframe2#100]
In https://github.com/lcnetdev/marc2bibframe2/blob/master/xsl/ConvSpec-648-662.xsl

Change madsrdf:isMemberofMADSScheme to madsrdf:isMemberOfMADSScheme

See http://id.loc.gov/ontologies/madsrdf/v1.html#p_isMemberOfMADSScheme

```
               <madsrdf:isMemberofMADSScheme>
                <xsl:attribute name="rdf:resource"><xsl:value-of select="."/></xsl:attribute>
              </madsrdf:isMemberofMADSScheme>
```

**Comment by kirkhess** [comment]
Also:
https://github.com/lcnetdev/marc2bibframe2/blob/4f3fcd3aa35280f1723a23cc5e216267e57bfffd/test/ConvSpec-1XX%2C6XX%2C7XX%2C8XX-names.xspec

https://github.com/lcnetdev/marc2bibframe2/blob/4f3fcd3aa35280f1723a23cc5e216267e57bfffd/test/marc2bibframe2.xspec

https://github.com/lcnetdev/marc2bibframe2/blob/4f3fcd3aa35280f1723a23cc5e216267e57bfffd/xsl/ConvSpec-1XX%2C6XX%2C7XX%2C8XX-names.xsl

https://github.com/lcnetdev/marc2bibframe2/blob/4f3fcd3aa35280f1723a23cc5e216267e57bfffd/xsl/ConvSpec-240andX30-UnifTitle.xsl

**Comment by dbs** [comment]
See https://github.com/lcnetdev/marc2bibframe2/pull/103 for a fix, including fixes for the unit tests.

**Comment by wafschneider** [comment]
Fixed in v1.4.0 branch.
